libafl/examples/llmp_test/main.rs

```diff
@@ -66,11 +66,17 @@ fn large_msg_loop(port: u16) -> ! {
         llmp::LlmpClient::create_attach_to_tcp(StdShMemProvider::new().unwrap(), port).unwrap();
 
     #[allow(clippy::large_stack_arrays)]
+    #[cfg(not(target_vendor = "apple"))]
     let meg_buf = [1u8; 1 << 20];
+    #[cfg(target_vendor = "apple")]
+    let meg_buf = [1u8; 1 << 19];
```

Comment:
Is there a reason why this has to be a stack allocation? `vec![1u8, 1<<20]` should be fine (it doesn't look like this example targets no_std)..

Reply:
I simply didn't see a reason not to use the stack here, 1 constant meg should fit, but you are right a vec cannot hurt either